crypto-ffi/src/generic/mod.rs

```diff
@@ -117,6 +117,16 @@ pub enum MlsError {
     /// with an External Commit instead
     #[error("Although this Welcome seems valid, the local KeyPackage it references has already been deleted locally. Join this group with an external commit")]
     OrphanWelcome,
+    /// This error is returned based on the abort reason returned in `MlsTransportResponse`. It is constructed when the reason is equal to `mls-client-mismatch`.
```

Comment:
Why don't make this decision though, the clients parse the backend error `mls-client-mismatch` and tells us to abort the operation? In theory there could be other cases where a client decides to abort an operation.

Reply:
Like for example because they reached the max amounts of retries.